.github/workflows/publish-maven-npm.yml

```
@@ -33,6 +33,15 @@ on:
           - io/inji
           - io/mosip
 
+      group_path:
```

actionlint (1.7.11)

[error] key "group_path" is duplicated in "inputs" section. previously defined at line:27,col:7. note that this key is case insensitive

(syntax-check)